### PR TITLE
Fix double word

### PR DESCRIPTION
**Pull request by bentley**

### CODE REVIEW

File: src/epub/text/chapter-9.xhtml [hunk 1]
```diff
@@ -17,7 +17,7 @@
...
+				<p>The actual and implicit practice and the conscious or stated aim thus balance each other. At different times such aims as complete living, better methods of language study, substitution of things for words, social efficiency, personal culture, social service, complete development of personality, encyclopedic knowledge, discipline, a esthetic contemplation, utility, <abbr>etc.</abbr>, have served. The following discussion takes up three statements of recent influence; certain others have been incidentally discussed in the previous chapters, and others will be considered later in a discussion of knowledge and of the values of studies. We begin with a consideration that education is a process of development in accordance with nature, taking Rousseau’s statement, which opposed natural to social (See <a href="chapter-7.xhtml#chapter-7-4-p-1">chapter 7</a>); and then pass over to the antithetical conception of social efficiency, which often opposes social to natural.</p>
```
Comment (acabal) [inline src/epub/text/chapter-9.xhtml]: Let's leave the ref as "here" as that's what we typically change refs to when they used to refer to printed page numbers.

**Review by acabal — CHANGES_REQUESTED** [review]

File: src/epub/text/chapter-9.xhtml (hunk 1 above)
Comment (bentley) [reply src/epub/text/chapter-9.xhtml]: Hm. I get that, but it means in two not unusual situations (printing out, or copy/pasting and losing the hyperlink) the reference becomes totally useless, where a chapter reference would retain some benefit.

I’ve also heard of it as an accessibility issue: a common screen reader operation is to gather a list of anchors and read them out so one can be selected. ([1](https://webaccess.berkeley.edu/ask-pecan/click-here), [2](https://usability.yale.edu/web-accessibility/articles/links))

Comment (acabal) [reply src/epub/text/chapter-9.xhtml]: Maybe, but if we do that we would have to change the whole corpus. Better to be consistent for now and we can revisit this as a corpus wide proposal later.

Comment (bentley) [reply src/epub/text/chapter-9.xhtml]: That’s fair. Updated.

**Comment by acabal** [comment]
Thanks!

**Pull request merged**